cmd/flux/diff_artifact.go

```diff
@@ -103,10 +113,112 @@ func diffArtifactCmdRun(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	if err := ociClient.Diff(ctx, url, diffArtifactArgs.path, diffArtifactArgs.ignorePaths); err != nil {
+	diff, err := diffArtifact(ctx, ociClient, url, diffArtifactArgs.path)
+	if err != nil {
 		return err
 	}
 
-	logger.Successf("no changes detected")
-	return nil
+	if diff == "" {
+		logger.Successf("no changes detected")
+		return nil
+	}
+
+	if !diffArtifactArgs.brief {
+		fmt.Print(diff)
```

Comment:
This should be `cmd.Print` like in `flux diff kustomization`

Reply:
✅ done